### PR TITLE
added warning for outdated policy

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #2272 ,
source [Security baseline for Windows 10 “April 2018 Update”](https://blogs.technet.microsoft.com/secguide/2018/04/30/security-baseline-for-windows-10-april-2018-update-v1803-final)

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/policy-csp-localpoliciessecurityoptions.md [hunk 1]
```diff
@@ -1587,6 +1587,10 @@ GP Info:
 
 <!--/Scope-->
 <!--Description-->
+
+> [!WARNING]
+> Starting in the version 1803 of Windows, this policy is deprecated.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-localpoliciessecurityoptions.md]:
```suggestion
> Starting in the Windows version 1803, this policy is deprecated.
```

Comment (MaratMussabekov) [reply windows/client-management/mdm/policy-csp-localpoliciessecurityoptions.md]: Dear @JohanFreelancer9 ,
would it be ok to change it to "Starting in Windows 10, version 1803, this policy is deprecated."?
Thank you

Comment (JohanFreelancer9) [reply windows/client-management/mdm/policy-csp-localpoliciessecurityoptions.md]: Dear @MaratMussabekov,
That would be perfect, thanks.

**Comment by mypil** [comment]
@ManikaDhiman - PR has been copy edited and is ready for final review. This is based on issue #2272.

Thank you.

**Pull request merged**